ros/src/control/scripts/thrust_control.py

```diff
@@ -3,33 +3,94 @@
 from shared_msgs.msg import can_msg, auto_control_msg, thrust_control_msg, thrust_status_msg, thrust_command_msg
 from sensor_msgs.msg import Imu
 from std_msgs.msg import Float32
+from rov import movement/mapper/Complex
+
+desired_a = None
+desired_p = None
+locked_dims_list = None
+disabled_list = None
+inverted_list = None
+auto_ctrl_flag = None
 
 def message_received(msg):
```

Comment:
Change names of message_received and command_received functions to be more obvious whether they are for auto or pilot/manual control.